modules/node/handler/page_builder.e

```diff
@@ -0,0 +1,70 @@
+note
+	description: "Generic Page Builder Interface"
+	date: "$Date$"
+	revision: "$Revision$"
+
+deferred class
+	PAGE_BUILDER [G->CMS_NODE]
```

Comment:
in there any reason to constraint G to CMS_NODE ?
and inherit from CMS_NODE_HANDLER ?

I am asking, in case we could reuse to display  list of .... users, list of ... anything